includes/REST_API/Stories_Media_Controller.php

```diff
@@ -207,7 +206,7 @@ protected function prepare_items_query( $prepared_args = [], $request = null ) {
 	 * @param WP_REST_Request $request Request object.
 	 * @return WP_REST_Response Response object.
 	 */
-	public function prepare_item_for_response( $post, $request ) {
+	public function prepare_item_for_response( $post, $request ): WP_REST_Response {
```

Comment:
The filter at the end means that this method could return something other than `WP_REST_Response`

Reply:
Hmm right